the-journals-of-gerontology-series-a.csl

```diff
@@ -0,0 +1,260 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US" page-range-format="expanded">
+  <info>
+    <title>The Journals of Gerontology: Series A</title>
+    <title-short/>
+    <id>http://www.zotero.org/styles/the-journals-of-gerontology-series-a</id>
+    <link href="http://www.zotero.org/styles/the-journals-of-gerontology-series-a" rel="self"/>
+    <link rel="template" href="http://www.zotero.org/styles/american-medical-association"/>
+    <link href="http://www.oxfordjournals.org/our_journals/gerona/for_authors/general.html" rel="documentation"/>
+    <link href="http://westlibrary.txwes.edu/sites/default/files/pdf/ama_citation_style.pdf" rel="documentation"/>
+    <author>
+      <name>Ulrik Stervbo</name>
+      <email>ulrik.stervbo@gmail.com</email>
+    </author>
+    <contributor>
+      <name>Julian Onions</name>
+      <email>julian.onions@gmail.com</email>
+    </contributor>
+    <contributor>
+      <name>Christian Pietsch</name>
+      <uri>http://purl.org/net/pietsch</uri>
+    </contributor>
```

Comment:
Please remove both contributors. The "template" link already takes care of attribution.